smartsim/_core/control/controller.py

```diff
@@ -70,12 +80,55 @@
     from ..launcher.stepMapping import StepMap
     from ..utils.serialize import TStepLaunchMetaData
 
+    _TELEMETRY_MONITOR: t.Optional[subprocess.Popen[t.Any]] = None
```

Comment:
I'm like 95% confident that all of our `Popen`s are `str` based, if you want to tighten up this typehint

Reply:
Fixed!